scripts/deploy-kit.js

```diff
@@ -1,49 +1,39 @@
 const deployDAOFactory = require('@aragon/os/scripts/deploy-daofactory.js')
-const KreditsKit = artifacts.require('KreditsKit')
 
 const fs = require('fs');
 const path = require('path');
 const libPath = path.join(__dirname, '..', 'lib');
 const addressesPath = path.join(libPath, 'addresses');
 
+const KreditsKit = artifacts.require('KreditsKit')
+
 const ensAddr = process.env.ENS
+const daoFactoryAddress = process.env.DAO_FACTORY
 
-module.exports = async (callback) => {
+module.exports = async function(callback) {
```

Comment:
Are you a javascript pro now?

Reply:
a pro in being desperate :) 
those scripts are executed in some truffle context and debugging sometimes is a pain because truffle swallows the errors and traces... so I sometimes threw everything on it :D

Reply:
Much better with a normal function!